contributors.yaml

```diff
@@ -83,5 +83,5 @@ contributors:
   -
     family-names: Chandra
     given-names: Abhavya
-    affiliation: "India Institute of Technology, Gandhinagar"
+    affiliation: "Indian Institute of Technology, Gandhinagar"    
```

Comment:
Remove the trailing whitespace here -- and configure your editor to trim trailing whitespace when saving files!

Reply:
removed trailing white-space,  sorry for the inconvenience.  And configured my editor to trim them.😅

Reply:
Thanks!